src/redux/sagas/actions/managePermissions.ts

```diff
@@ -94,6 +111,28 @@ function* managePermissionsAction({
       );
     }
 
+    yield put(transactionPending(setUserRoles.id));
```

Comment:
Any reason to prefer doing it like this instead of moving the logic up and adding the params directly?

Reply:
Consistency mostly, as for some sagas this is actually required since you don't have the params available when creating the transaction channel _(ie: you need data from the first transaction that gets executed)_

The second reason, is UX. For sagas/transactions that require us computing their proofs, on slower machines it might take some time, so I've chose this pattern since `transactionPending` will start showing the spinner next to the transaction (in the gas station), while we compute data, and send it, in the background.

Otherwise, the user will just sit there with a transaction entry with no state.